CHANGELOG.md

```diff
@@ -64,6 +64,12 @@ This project tries to follow [SemVer 2.0.0](https://semver.org/).
   ID has not been shown to have any benefits. Please refer to the
   `GET /project/{projectId}` endpoint instead. (#75)
 
+- Added `TestResultListSummary` field to `Build` database model. This should
+  allow us to avoid `N+1 Select` in wharf-api. (#80)
```

Comment:
The implementation in `getBuilds` inside `project.go` still is a `N+1` implementation for the SQL SELECTs.

I'd say it's OK for now, especially as users typically query at most 10 builds at a time, but I'd suggest changing this phrasing

```suggestion
- Added `TestResultListSummary` field to `Build` database model. This allow you
  to avoid `N+1` HTTP requests when listing builds to show test summaries. (#80)
```